lib/ansible/modules/packaging/os/pkgutil.py

```diff
@@ -27,30 +27,43 @@
     - Pkgutil is an advanced packaging system, which resolves dependency on installation.
       It is designed for CSW packages.
 version_added: "1.3"
-author: "Alexander Winkler (@dermute)"
+author:
+    - "Alexander Winkler (@dermute)"
+    - "David Ponessa (github.com/scathatheworm)"
```

Comment:
Replace with `David Ponessa (@scathatheworm)`

The @githubid is expected not @twitterid